src/Http/Livewire/CKEditor.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
if we could have something like `onCellClick` you could call `Livewire.emit('openModal', 'laravel-views-ckeditor')` from there to open up this modal even if the modals package is not supported out of the box, that wouldn matter, you could you whatever you need when a cell was clicked

Reply:
I’m not sure onCellClick will be a good idea.
Now we can have 2 or more clicking zones inside a cell, onCellClick will have just one

Reply:
I liked your idea better about having some sort of `UI::actionButton()` method/component.